index.html

```diff
@@ -279,5 +279,33 @@ <h3 class="titleA">About My self</h3>
                 </li>
               </ul>
             </section>
-  
-  
\ No newline at end of file
+            <!--Works Section contact page-->
+            <section class="page5">
+              <div class="@continer">
+                <div class="bag">
+                <li class="shape1"><img src="image/background/contact-top1.png"></li>
+                <li class="shape2"><img src="image/background/contact-bottom.png"></li>
+                <li class="shape3"><img src="image/background/Capture3.PNG"></li>
+                <div class="h1">
+                  <li class="h2"> <h2>Contact me</h2></li>
+                  <li class="p1"> <p>If you have an application you<br>are interested in developing, a feature that you need built or a project that needs coding.<br> I’d love to help with it!</p></li>
```

Comment:
- [ ] The `<li>` tags should always be used inside `<ul>` tags. There role is to make items as part of a list. Their use here is not right since they are not in a `<ul>` and the the items also do not belong together. 🥇